lib/linter-integration.js

```diff
@@ -0,0 +1,31 @@
+'use babel';
+
+export default class LinterIntegration {
+  constructor(linterInstance) {
+    this.linter = linterInstance;
+    linterInstance.addLinter(this.provider);
+  }
+
+  get provider() {
```

Comment:
This creates a new object everytime you access it, it'll confuse the base linter and result in duplicate messages. You should always use the **same** object.